Assets/MRTK/SDK/Experimental/Elastic/Scripts/LinearElasticSystem.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+
+[assembly: InternalsVisibleTo("Microsoft.MixedReality.Toolkit.Tests.PlayModeTests")]
+namespace Microsoft.MixedReality.Toolkit.Experimental.Physics
+{
+    public class LinearElasticSystem : ElasticSystem<float>
```

Comment:
This elastic system already existed in the prior version of the Elastic classes that were already merged in; this particular one used to be located in `ElasticSystem.cs` but is now in its own file, for consistency.